### PR TITLE
10 sale exception mig

### PR DESCRIPTION
**Pull request by mourad-ehm**
Add openupgrade migration script.

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
LGTM

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@mouadhamri do you still consider this as WIP?

**Comment by mourad-ehm** [comment]
No

File: sale_exception/migrations/10.0.2.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    # Update exception_rule for sale.order and sale.order.line:
+    cr.execute(
```
Comment (pedrobaeza) [inline sale_exception/migrations/10.0.2.0.0/pre-migration.py]: I think it's better to use `openupgrade.logged_query` for logging the number of results and so on.

Comment (mourad-ehm) [reply sale_exception/migrations/10.0.2.0.0/pre-migration.py]: cr.execute replaced by openupgrade.logged_query.

**Review by bealdav — APPROVED** [review]

**Pull request merged**